main.rb

```diff
@@ -45,10 +45,10 @@ def get_stats
 def get_player_choice
   print "(A)ttack, (D)efend, or (F)lee? "
 
-  case gets.chomp.first
-    when "A" then return "attack"
-    when "D" then return "defend"
-    when "F" then return "flee"
+  case gets.chomp
+    when /\Aa/i then return "attack"
```

Comment:
What I'm doing here is forcing the match to start at the beginning of the string and not check if it is upper or lower case.